src/main/java/graphql/parser/Parser.java

```diff
@@ -46,6 +49,11 @@
 @PublicApi
 public class Parser {
 
+    @Internal
+    public static final int CHANNEL_COMMENTS = 2;
+    @Internal
```

Comment:
lets call this also whitespace channel and not ignored ones to make it consistent with the options.